understanding/page-titled.html

```diff
@@ -58,121 +56,94 @@
                   
                </p>
-               <p>In cases where the page is a document or a web application, the name of the document
-                  or web application would be sufficient to describe the purpose of the page. Note that
-                  it is not required to use the name of the document or web application; other things
-                  may also describe the purpose or the topic of the page. 
+               <p>ページが文書又はウェブアプリケーションの場合、文書又はウェブアプリケーションの名称はページの目的を説明するのに十分だろう。ここで注意すべきなのは、文書又はウェブアプリケーションの名称を使用することを求められていないことである。他の要素によってページの目的又は主題が説明される場合もあるだろう。
                   
                </p>
-               <p>In cases such as Single Page Applications (SPAs), where various distinct pages/views
-                  are
-                  all nominally served from the same URI and the content of the page is changed dynamically,
-                  the title of the page should also be changed dynamically to reflect the content or
-                  topic of
-                  the current view.
-                  
+               <p>シングルページアプリケーション (SPA) のように、異なるページ又はビューが名目上、全て同じ URI から提供され、ページのコンテンツが動的に変更される場合、ページのタイトルも、現在のビューのコンテンツ又はトピックを反映して動的に変更されるべきである。
                </p>
                <p>
-                  
-                  <a href="link-purpose-in-context" class="understanding">Success Criteria 2.4.4</a> and 
-                  <a href="link-purpose-link-only" class="understanding">2.4.9</a> deal with the purpose of links, many of which are links to web pages. Here also,
-                  the name of a document or web application being linked to would be sufficient to describe
-                  the purpose of the link. Having the link and the title agree, or be very similar,
-                  is good practice and provides continuity between the link 'clicked on' and the web
-                  page that the user lands on. 
+                  <a href="link-purpose-in-context" class="understanding">達成基準 2.4.4</a> 及び <a href="link-purpose-link-only" class="understanding">2.4.9</a> はリンクの目的を取り扱い、その多くはウェブページへのリンクである。ここでも、リンクされた文書又はウェブアプリケーションの名称がリンクの目的を説明するのに十分だろう。リンクとタイトルが一致する又は極めて似ていることは、グッドプラクティスであり、リンクの「クリック」と利用者が着地するウェブページとの間の連続性を提供する。
                   
                </p>
             </section>
             <section id="benefits">
-               <h2>Benefits</h2>
+               <h2>利点</h2>
                <ul>
                   
                   
                   <li>
-                     This criterion benefits all users in allowing users to quickly and easily identify
-                     whether the information contained in the Web page is relevant to their needs. 
+                     この達成基準は、そのウェブページにある情報が自分のニーズに関係があるかどうかを、すべての利用者が素早くかつ容易に確認できるようにする。
```

Comment:
「すべての利用者が素早く……」というのは微妙に違っているような。「利用者が素早く……を許すことで全ての利用者に利益をもたらす」という文だと思います。こうでしょうか?

```suggestion
                     この達成基準は、そのウェブページにある情報が利用者のニーズに関係があるかどうかを素早くかつ容易に確認できるようにすることで、すべての利用者に利益をもたらす。
```

Reply:
ありがとうございます。　改めて見ると、benefits がうまく訳出されていませんでしたね...。